cypress/e2e/right-panel/file-panel.spec.ts

```diff
@@ -132,14 +135,46 @@ describe("FilePanel", () => {
                         cy.contains(".mx_MFileBody_info_filename", "matrix-org");
                     });
                 });
+            });
+
+            // Make the viewport tall enough to display all of the file tiles on FilePanel
+            cy.viewport(660, 1000);
```

Comment:
It is required to check visibility of all of the file tiles. Otherwise it will return an error on the default viewport size `(1000, 600)`.